src/stylesheets/base/_typography.scss

```diff
@@ -1,19 +1,10 @@
-@font-face {
-    font-family: geomanist;
-    src: url(fonts/geomanist-regular-webfont.woff2);
-}
-
-@font-face {
-    font-family: geomanist-medium;
-    src: url(fonts/geomanist-medium-webfont.woff2);
-}
-
 body {
-  font-family: 'geomanist';
-  font-size: 16px;
-  color: #504949;
+  color: $florence-gray;
+  font-family: $main-font, sans-serif;
```

Comment:
I'd move `sans-serif` to your `$main-font` variable (in _vars.scss), e.g. `$main-font: geomanist, sans-serif;`. Also, you won't need quotes around font names unless they contain whitespace or weird characters.